server/doc.js

```diff
@@ -0,0 +1,34 @@
+const mongoose = require('mongoose');
+mongoose.promise = Promise;
+const Schema = mongoose.Schema;
+const date = require('date-and-time');
+
+const docSchema = new Schema({
+    name: {type: String, required = true},
+    file: {type: String, required = true},
+    owner_id: {type: int, required = true},
+    locked: int,
+    revision_id: int,
+    last_revised: {type: Date, required = true}
+})
+
+docSchema.methods.filter = function(file_text, taboo_words){
+    //A String is considered a word if there is a space
+    //Checks all the words and see if any are in taboo words and replaces those words
+    let wordsarr = file_text.split(" ");
```

Comment:
If you're going to go with this approach, you have to join the array back afterwards and reassign it to this.file

Reply:
Joined the split and changed array back together.